lib/foreman_theme_satellite/branded_words.rb

```diff
@@ -37,13 +37,13 @@ module ForemanThemeSatellite
     /\bSmart-Proxies\b(?!-)/ => 'Capsules',
     /\bSmart proxy\b(?!-)/   => 'Capsule',
     /\bSmart-Proxy\b(?!-)/   => 'Capsule',
-    /\boVirt\b(?!-)/         => 'RHEV',
-    /\bovirt\b(?!-)/         => 'RHEV',
-    /\bOvirt\b(?!-)/         => 'RHEV',
+    /\boVirt\b(?!-)/         => 'RHV',
+    /\bovirt\b(?!-)/         => 'RHV',
+    /\bOvirt\b(?!-)/         => 'RHV',
```

Comment:
While we're cleaning up, can't we just make it case insensitive and always replace it? I think this should work:
```suggestion
    /\boVirt\b(?!-)/i        => 'RHV',
```

Reply:
I have taken the liberty of simplifying the regular expressions in general. A lot of them seemed redundant to me. 
I have also removed the option of branding to "**capsule**", as "**Capsule**" is always written with a capital C in the documentation. Furthermore, I also removed the branding of "**proxy**", as that is, IMHO, unclear language and should be treated as a bug like [here](https://github.com/Katello/katello/pull/11075).
I also added a unit test that tests the branding with strigs from the docs.
We are using a similar configuration to this with great success!